components/cstudio-forms/controls/rte.js

```diff
@@ -465,7 +465,7 @@ YAHOO.extend(CStudioForms.Controls.RTE, CStudioForms.CStudioFormField, {
 	        editor_selector : rteUniqueInitClass,
 	        theme : "advanced",
 	        skin : "cstudio-rte",
-	        width : width,
+			width: '100%', // Why? Field width should be flexible, because of responsiveness, should resize based on the screen size, this works together with max width set on post render event below on setup: method 
```

Comment:
well, width is defined by form definition, but if you said for example 800px, if you resize the window, the field will go out of the form, because width is overflowing the available container space, so fix was to make it 100% but use the definition width for max width